bundle/edu.gemini.too.event/src/main/scala/edu/gemini/too/event/service/TooCondition.scala

```diff
@@ -13,26 +14,36 @@ import edu.gemini.spModel.too.Too
  * observation's status must be transitioned from `ON_HOLD` to `READY` and be
  * a ToO observation.
  *
+ * (REL-566: skip acquisition and calibration observations as well.)
+ *
  * This condition is registered with the database such that when it occurs, the
  * [[edu.gemini.too.event.service.TooService]] is executed to record the event.
  */
 object TooCondition extends IDBTriggerCondition {
-  private def isDataObjectUpdate(change: SPCompositeChange): Boolean =
-    change.getPropertyName == SPUtil.getDataObjectPropertyName
+  def triggeredObservation(change: SPCompositeChange): Option[ISPObservation] = {
+    def castIf[T:Manifest](o: Object): Option[T] =
```

Comment:
Why don't we just cast it without checking?  The API lets you store whatever you want as a data object of any program node, but if we do that everything will break long before getting to ToO triggers anyway.